### PR TITLE
task/handle-data-offload-options

### PR DESCRIPTION
**Pull request by ferrom-jaia**
# Summary
Adding config for data offload type

# Details
The approach for the data offload type is similar to the imu type configuration

# Testing

This will need to be tested on a bot after merging in. 

# Notes
Please merge this pull request in first: https://github.com/jaiarobotics/jaiabot-debian/pull/54

### CODE REVIEW

File: config/gen/bot.py [hunk 1]
```diff
@@ -55,6 +55,16 @@
 else:
     jaia_arduino_dev_location="/dev/ttyAMA1"
```
Comment (michael-jaia) [inline config/gen/bot.py:57]: Do we need the duplicate checks if we just pass in GOBY, TASKPACKET, or NONE

**Review by michael-jaia — COMMENTED** [review]
Same as last PR, not sure if it's necessary, but thought I'd leave the comment just in case. Also why do we need to adjust the both gen/bot.py and gen/systemd.py?

**Comment by ferrom-jaia** [comment]
> Same as last PR, not sure if it's necessary, but thought I'd leave the comment just in case. Also why do we need to adjust the both gen/bot.py and gen/systemd.py?

We need systemd.py so our services get the correct configurations and it is also responsible for making the configurations environmental variables. We need the change in bot.py to update the mission manager configuration with the correct value from the environmental variable.

**Review by michael-jaia — APPROVED** [review]
Comments resolved, looks good, approving PR into 1.y

**Pull request merged**